### ISSUE TITLE
Standard HeSuVi HRIR speaker arrangement

### ISSUE THREAD

**Issue by ThreeDeeJay** [davircarvalho/Individualized_HRTF_Synthesis#10]
tl;dr my suggestion is simply setting front speakers' default azimuth to 30° since that seems to be the common denominator in standards. Seems like a nice simple edit that'd be nice for the next update 👀👌

Currently, the HeSuVi speaker azimuths for Front/Side/Rear are 35°/110°/150°, but I'm not sure that's a standard.
![image](https://user-images.githubusercontent.com/71472458/220634812-8ecf95f1-1aac-423f-9000-2ec022259830.png)

The [HeSuVi developer prefers 30°/90°/135°](https://i.imgur.com/qB8ZaK9.png), which is a (if not *the*) gaming standard:
![image](https://user-images.githubusercontent.com/71472458/220633695-67c2a137-6108-4585-8fca-35d8c3801610.png)
https://etheses.whiterose.ac.uk/23687/14/Thesis.pdf#page=117
https://www.itu.int/dms_pubrec/itu-r/rec/bs/R-REC-BS.775-4-202212-I!!PDF-E.pdf#page=5

I've also seen 22.5°([movies](https://en.wikipedia.org/wiki/5.1_surround_sound#Music))-30°([music](https://en.wikipedia.org/wiki/5.1_surround_sound#Music))/90°-110°/135°-150°
[![IMAGE ALT TEXT](https://www.audioholics.com/audio-technologies/7-1-surround-sound/image)](https://www.audioholics.com/audio-technologies/7-1-surround-sound/image "7.1 surround")

Ideally, we'd always have the full 3D HRTF at one's disposal and virtual speakers would match the exact angles the mix was made for so we can retain positional accuracy, but surround speaker angles are rarely documented in games. Other than the example above, I just know about source games using 45°/45°([front repeated for "wide" configuration](https://i.imgur.com/VbjDcdN.mp4))/135° which completely ignores standards. Perhaps there's a way to dig into popular sound engines like Unreal/Unity (or just the documentation) to see what angles they use 🤔

**Comment by davircarvalho** [comment]
default speaker setup was changed in the latest release

**Comment by ThreeDeeJay** [comment]
Awesome, thanks! 👏
